docs/articles_en/documentation/openvino-extensibility/custom-openvino-operations.rst

```diff
@@ -7,30 +7,64 @@ Custom OpenVINO Operations
                  custom operations to support models with operations
                  not supported by OpenVINO.
 
-OpenVINO™ Extension API allows you to register custom operations to support models with operations which OpenVINO™ does not support out-of-the-box. This capability requires writing code in C++, so if you are using Python to develop your application you need to build a separate shared library implemented in C++ first and load it in Python using ``add_extension`` API. Please refer to :ref:`Create library with extensions <create_a_library_with_extensions>` for more details on library creation and usage. The remaining part of this document describes how to implement an operation class.
+OpenVINO™ Extension API allows you to register custom operations to support models with operations which OpenVINO™ does not support out-of-the-box.  A custom operation might be implemented both in C++ and Python.```
+
+Also it is possible to create a shared library with custom operation implemented in C++ first and load it in Python using ``add_extension`` API. Please refer to :ref:`Create library with extensions <create_a_library_with_extensions>` for more details on library creation and usage. The remaining part of this document describes how to implement an operation class using both the C++ API and Python API.
```

Comment:
The meaning was changed. In the old piece there is a context: "if you are using Python". Here there is no such context, so it looks like `add_extension` is used in Python only. `add_extension` can be used in both C++ and Python, there is no bias.

Reply:
Actually this method is also available in JS API, so I think I can omit mentioning the language here altogether